sdk/src/main/scala/io/horizen/account/api/rpc/service/RpcProcessor.scala

```diff
@@ -0,0 +1,63 @@
+package io.horizen.account.api.rpc.service
+
+import com.fasterxml.jackson.databind.JsonNode
+import io.horizen.account.api.rpc.handler.{RpcException, RpcHandler}
+import io.horizen.account.api.rpc.request.{RpcId, RpcRequest}
+import io.horizen.account.api.rpc.response.RpcResponseError
+import io.horizen.account.api.rpc.utils.{RpcCode, RpcError}
+import io.horizen.account.serialization.EthJsonMapper
+import sparkz.util.SparkzLogging
+
+import scala.jdk.CollectionConverters.asScalaIteratorConverter
+import scala.util.{Failure, Success, Try}
+
+
+object RpcProcessor extends SparkzLogging {
```

Comment:
I still vote to keep `RpcProcessor` as a class, not as an object.
In the AccountSidechainApp create an instance of this class and then pass as an argument of:
1) `AccountEthRpcRoute` constructor, to let this class use RpcProcessor explicitly. Implicit dependencies are not good to have.
2) `WebSocketAccountServerRef` constructor, and then inside do the same trick as for NVHactor.

Reply:
Done